coalib/settings/Setting.py

```diff
@@ -44,6 +45,21 @@ def glob_list(obj, *args, **kwargs):
     return obj.__glob_list__(*args, **kwargs)
 
 
+def language_obj(name):
```

Comment:
I'm not keen on this name.  This name needs to be used in `Bear.run()` override arg lists, and becomes part of our documentation.  Can it simply be `language` ?  And be used as `def run(.... , language: language):` ; while that looks like it might be wrong, it should work and do the right thing.

The other alternative is that the `run` argument validator should work correctly with `def run(.... , language: Language):` , which would require improvements to the argument validator so that dict subclass, like `Language`, are checked using the value to key-index the dict, and thus `KeyError` would be the expected exception of incorrect values instead of `ValueError`.

Also, in order to properly solve https://github.com/coala/coala/issues/4463 , this PR should include a test which is a Bear using `language: language`.